Tutorial button hidden on login screens

On all touch devices, the tutorial button is hidden and the entire UI is shifted upwards on login/create user screens. See here:

![loginHelpIssue](https://user-images.githubusercontent.com/40134655/76633646-0d768a80-656b-11ea-8112-c24d297aac03.gif)

This was the case for existing users. The issue can also be replicated for a new user if the login screen is refreshed once.

![loginHelpIssue2](https://user-images.githubusercontent.com/40134655/76634249-0c922880-656c-11ea-80b8-2e424f8591db.gif)

I'm working on a fix.

I'm encountering an error while building the apk
```
FAILURE: Build failed with an exception.

* What went wrong:
Gradle build daemon disappeared unexpectedly (it may have been killed or may have crashed)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
Error: /sugarizer-cordova/platforms/android/gradlew: Command failed with exit code 1 Error output:
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
The message received from the daemon indicates that the daemon has disappeared.
Build request sent: Build{id=937565a5-5dae-424d-a577-b878e2a59ae6.1, currentDir=/sugarizer-cordova}
Attempting to read last messages from the daemon log...
Daemon pid: 1310
  log file: /root/.gradle/daemon/3.3/daemon-1310.out.log
----- Last  20 lines from daemon log file - daemon-1310.out.log -----
:incrementalDebugJavaCompilationSafeguard
:compileDebugJavaWithJavac
:compileDebugJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
:compileDebugNdk UP-TO-DATE
:compileDebugSources
:mergeDebugShaders
:compileDebugShaders
:generateDebugAssets
:mergeDebugAssets
:transformClassesWithDexForDebug
:mergeDebugJniLibFolders
:transformNative_libsWithMergeJniLibsForDebug
:processDebugJavaRes UP-TO-DATE
:transformResourcesWithMergeJavaResForDebug
:validateSigningDebug
:packageDebug
----- End of the daemon log -----
```

--- a/css/styles.css
+++ b/css/styles.css
@@ -1138,7 +1138,6 @@ body {
 
 .first-namebox {
 	height: 100%;
-	padding-bottom: 100%;
 }
 
 .first-nameline {
@@ -1159,7 +1158,6 @@ body {
 
 .first-serverbox {
 	height: 100%;
-	padding-bottom: 100%;
 }
 
 .first-serverline {
@@ -1253,7 +1251,6 @@ body {
 
 .first-passbox {
 	height: 100%;
-	padding-bottom: 100%;
 }
 
 .first-input {

--- a/css/styles.css
+++ b/css/styles.css
@@ -1125,7 +1125,7 @@ body {
 }
 
 .first-help {
-	position: absolute;
+	position: fixed;
 	top: 10px;
 	left: 30px;
 }
@@ -1138,6 +1138,7 @@ body {
 
 .first-namebox {
 	height: 100%;
+	padding-bottom: 100%;
 }
 
 .first-nameline {
@@ -1158,6 +1159,7 @@ body {
 
 .first-serverbox {
 	height: 100%;
+	padding-bottom: 100%;
 }
 
 .first-serverline {
@@ -1251,6 +1253,7 @@ body {
 
 .first-passbox {
 	height: 100%;
+	padding-bottom: 100%;
 }
 
 .first-input {

